docs/core/diagnostics/collect-dumps-crash.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Collect Dumps on Crash - .NET
+description: How to collect a dump on a crash
+ms.date: 12/13/2022
+---
+
+# Collect dumps on crash
+
+Configuring your application to collect a dump on crash is done by setting specific environment variables. This is helpful when you want to get an understanding of why a crash happened. For example, capturing a dump when an exception is thrown helps you identify an issue by examining the state of the app when it crashed.
+
+The following table shows the environment variables you can configure for collecting dumps on a crash.
+
+|Environment variable|Description|Default value|
+|-------|---------|---|
+|`COMPlus_DbgEnableMiniDump` or `DOTNET_DbgEnableMiniDump`|If set to 1, enable core dump generation.|0|
+|`COMPlus_DbgMiniDumpType` or `DOTNET_DbgMiniDumpType`|Type of dump to be collected. For more information, see the table below|2 (`MiniDumpWithPrivateReadWriteMemory`)|
```

Comment:
```suggestion
|`COMPlus_DbgMiniDumpType` or `DOTNET_DbgMiniDumpType`|Type of dump to be collected. For more information, see [Types of mini dumps](#types-of-mini-dumps).|2 (`MiniDumpWithPrivateReadWriteMemory`)|
```

Reply:
I'm also confused why 2 corresponds to `MiniDumpWithPrivateReadWriteMemory` here, but in the referenced table, it corresponds to `Heap`. Are there two different ways to write it?

Reply:
@noahfalk would you please lend some insight to this?

Reply:
Those two names are refering to the same concept and I'd recommend we replace MiniDumpWithPrivateReadWriteMemory -> Heap for naming consistency.

I believe what happened is in the past the doc used those very long names everywhere which correspond to some constants defined on the win32 MiniDumpWriteDump API. However someone probably recognized there was little value in using complicated API naming details when explaining these concepts. When they refactored to simpler names this reference probably got overlooked.